### DISCUSSION TITLE
SkyWalking installation

### DISCUSSION THREAD

**Discussion by Donato29** [apache/skywalking#9346] [General]
I am new to Skywalking . I want to install Skywalking for monitoring our server but i want to try it first at my local.

**Comment by Superskyyy** [comment]
Hello there, if you want to feel it's true power you will need a complete showcase, refer to this guide. The below showcase documentation will give you the same stuff we have on the [demo site here](http://demo.skywalking.apache.org) (user:skywalking, password:skywalking)

https://skywalking.apache.org/docs/skywalking-showcase/latest/readme/

Otherwise, trying the very basics without Docker/K8s is also super easy. Simply download the v9 binary release and run the start script in /bin and voilà there you have it. Access the UI from localhost:8080. Note this way you need to provide your own service to monitor. The above showcase takes care of everything for you.

**Comment by Superskyyy** [comment]
After you played with the showcase, please start from the SkyWalking Documentation site to consider for production deployment, it's all there.
